### PR TITLE
Milestone 1: Portfolio mobile skeleton

### PR DESCRIPTION
**Pull request by erickhansx**
Implemented the first part of the mobile version of the portfolio skeleton:

- Added the text content to the Index.html file.
- Added CSS file and styles to the content of the page.
- Divided the page into two main containers implementing the CSS technology Flexbox.
- Added the linters to the portfolio project.
- Added a descriptive README file utilizing the template provided.
- Implemented button interactions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
-# portfolio
\ No newline at end of file
+# ![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio Skeleton
+
+> This is my first portfolio.
+> It was based on a FIGMA template and I used HTML,
+> CSS and CSS FLEXBOX to build the first part of the mobile version.
+
```
Comment (nicupop729) [inline README.md:8]: In order to have a more professional repo, it is better if we don't exclude sections from README.md file.

- [x] Kindly, add back the following sections:
`Getting Started`
`Prerequisites` - what the user needs on his machine to reproduce your project
`Setup` - help with guided steps a non-professional user to clone your repo on his machine
`Install` - explain how  a non-professional user can run your project on his device

README Template can be found here: [Template](https://github.com/microverseinc/readme-template);

**After this, definitely, your repo will look professional** :1st_place_medal:

File: resources/css/style.css [hunk 2]
```diff
@@ -0,0 +1,103 @@
+/* stylelint-disable function-url-quotes */
+/* stylelint-disable font-family-name-quotes */
```
Comment (nicupop729) [inline resources/css/style.css]: It is recommended that we follow the good practices and we code following a code of conduct. For this reason, it is not advisable to deactivate linters.

- [x] Kindly, try to work around it without deactivating linters

File: resources/css/style.css [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+/* Global */
+
+html {
+  font-size: 62.5%; /* 1 REM = 10PX */
```
Comment (nicupop729) [inline resources/css/style.css]: Commented code makes your files look nasty, unorganised, and unprofessional.

- [ ] Please, remove the commented code from your files for the reasons mentions above :+1:

File: resources/css/style.css [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+}
+
+/* TOP NAV */
+
+.burger {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  /* outline: 1px solid blue; */
```
Comment (nicupop729) [inline resources/css/style.css]: Commented code makes your files look nasty, unorganised, and unprofessional.

- [ ] Please, remove the commented code from your files for the reasons mentions above :+1: 

- [ ] Kindly, remove the unnecessary empty spaces as well;

File: index.html [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <style>
+      @import url("https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;700;900&family=Poppins:wght@400;700;900&display=swap");
+    </style>
+    <style>
+      @import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap");
+    </style>
```
Comment (nicupop729) [inline index.html]: [OPTIONAL]

It is advisable to introduce the styles in our project using the CSS files and not thru HTML;

- [ ] Kindly, move the imported fonts to the CSS file;

File: index.html [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+  <body>
+    <main>
+      <header class="top-nav-burger">
+        <nav class="container burger">
+          <div class="top-corner-image">
+            <img
+              src="/resources/images/image_geometry_1.png"
+              alt="corner image"
+            />
+          </div>
```
Comment (nicupop729) [inline index.html]: As the logo is not part of the navigation, it is better and recommendable to have it outside of the `nav` tag, but still inside the `header` tag.

- [ ] Kindly, place the logo outside the `nav` tag but still inside the `header` :sun_with_face:

**Review by nicupop729 — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️

Hi @erickhansx, 

Good job so far! 
**You are doing a very nice project, with clean code and respecting Figma's design!**
There are some issues that you still need to work on to go to the next project but you are almost there!




### Some highlights for you
* Correct github flow practice ✔️ 
* Good commit messages ✔️ 
* Linters are passing :heavy_check_mark: 
* Clean codebase ✔️ 
* Figma design is respected :heavy_check_mark: 


### Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @erickhansx , 

Good job so far. I really like what you did!
There are some minor issues that you still need to work on to go to the next project but you are almost there!
mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
You implemented all the changes suggested by the previous reviewer✔️

## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve
  
 ### After checking the layout of your project, this is what I saw
  
image

* I realized that there was a problem with the link to your CSS files and your images. So, to fix that you can remove the `/` at the beginning of the references. Like replacing `/resources/...` with `resources/...`

_I'm sure you can do it just keep it up_
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @erickhansx , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR and Description

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me using @danmainah in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**